content/docs/intro/concepts/state.md

```diff
@@ -139,221 +98,167 @@ Enter your access token from https://app.pulumi.com/account/tokens
 
-### Checking your backend info
+# logout and login to the desired new backend
+$ pulumi logout
+$ pulumi login # default to Pulumi Service
 
-To quickly check your backend login info, run `pulumi whoami` with the `--verbose` (or `-v`) flag.
+# create a new stack with the same name on pulumi.com
+$ pulumi stack init my-app-production
 
-```bash
-$ pulumi whoami -v
+# import the new existing checkpoint into pulumi.com
+$ pulumi stack import --file my-app-production.checkpoint.json
 ```
 
-See [pulumi whoami]({{< relref "/docs/reference/cli/pulumi_whoami" >}}) for more details.
+After performing these steps, your stack will now be under the management of the Pulumi Service. All subsequent operations should be performed using this new backend.
 
-#### Example response
+> **Note:**: After migration, your stack's state will be managed by the the Pulumi Service backend, but the stack will continue using the same secrets provider.  You can separately [change the secrets provider]({{< relref "docs/intro/concepts/config#changing-the-secrets-provider-for-a-stack" >}}) for your stack if needed.
 
-```
-User: <your-username>
-Backend URL: https://app.pulumi.com/<your-username>
-```
+### Pulumi Service Architecture
 
-### Adopting the Pulumi Service backend
+This Pulumi Service is comprised of two Internet-accessible endpoints &mdash; a web application at `app.pulumi.com` (called [the "Pulumi Console"]({{< relref "/docs/intro/console" >}})) and a REST API at `api.pulumi.com` &mdash; with an assortment of cloud infrastructure to support its features. A simplified diagram of its architecture looks like this:
 
-If you are currently using a self-managed backend, but would now prefer to
-simplify things, just run `pulumi logout` and then `pulumi login`, and you’ll be back to
-using `app.pulumi.com`.
+<img src="/images/docs/reference/state_saas.png" alt="Pulumi Service Architecture" class="img-bordered">
 
-If you'd like to migrate your stacks from a self-managed backend to the Pulumi Service
-backend, you can follow the steps below. Suppose the stack "my-app-production"
-has been managed with a self-managed backend, and you want to migrate it to
-the Pulumi Service. Run the following commands if you are already logged in to the local
-endpoint:
+The Pulumi Service doesn't ever acquire your cloud credentials, and does not communicate with your cloud provider directly. Instead, the CLI itself coordinates with both the Pulumi Service's API and your cloud provider's directly. This ensures your IAM and key management does not need to change while adopting Pulumi. In particular, if you are running Pulumi deployments from [within a CI/CD environment]({{< relref "/docs/guides/continuous-delivery" >}}), you can rely on existing mechanisms and security practices that your organization has already put in place. The Pulumi Service is reliable, secure, and has undergone multiple audits, including SOC2 and professional pen-testing.
```

Comment:
"...and your cloud provider's..." wasn't very explicit, so I took a shot at filling in the missing details.

```suggestion
The Pulumi Service doesn't ever acquire your cloud credentials, and does not communicate with your cloud provider directly. Instead, the CLI itself coordinates with both the Pulumi Service's API and your cloud provider's API directly. This ensures your IAM and key management does not need to change while adopting Pulumi. In particular, if you are running Pulumi deployments from [within a CI/CD environment]({{< relref "/docs/guides/continuous-delivery" >}}), you can rely on existing mechanisms and security practices that your organization has already put in place. The Pulumi Service is reliable, secure, and has undergone multiple audits, including SOC2 and professional pen-testing.
```